dbms/src/AggregateFunctions/AggregateFunctionUniqCombined.h

```diff
@@ -85,9 +90,9 @@ template <typename Key>
 struct AggregateFunctionUniqCombinedDataWithKey<Key, 17>
 {
     using Set = CombinedCardinalityEstimator<Key,
-        HashSet<Key, TrivialHash, HashTableGrower<>>,
+        HashSet<Key, TrivialHash, UniqCombinedHashTableGrower>,
         16,
-        13,
+        12 + (sizeof(Key) == sizeof(UInt32)),
```

Comment:
Here we have lowered max size degree by one when type is UInt64 and keep as before for UInt32. It's Ok.

Reply:
This `+ (sizeof(Key) == sizeof(UInt32))` looks totally puzzling. Why not just write it out:
```
template <size_t size> constexpr int hll_threshold = 12;
template <> constexpr int hll_threshold<sizeof(UInt32)> = 13; 
```

Reply:
Well it doesn't look puzzlingly to me, but I agree with you.
However, this bit is in two places, and writing this in your way will overly complicate the code
Although I don't have any strong feeling about this and can go your way

Reply:
If I'm not missing anything, it should reduce duplication, you just write `hll_threshold<sizeof(Key)>` instead of `12 + sizeof...`.

Reply:
@akuzm Yes, it will be more clear. You can do this modification.